### PR TITLE
fix bob-core cleanup

### PR DESCRIPTION
**Pull request by azimut**
Missed backslashes and ignore error on find due missing dirs on image.

### CODE REVIEW

File: bob-core/build-root.sh [hunk 1]
```diff
@@ -390,17 +390,19 @@ if [ -z "$BOB_SKIP_BASELAYOUT" ]; then
 fi
 
 # clean up
+set +e
 find "${EMERGE_ROOT}"/lib64 "${EMERGE_ROOT}"/usr/lib64 \
     -type f \( -name '*.[co]' -o -name '*.prl' \) -delete
+set -e
```
Comment (jbergstroem) [inline bob-core/build-root.sh]: Why two?

Comment (azimut) [inline bob-core/build-root.sh]: the script has a "-e" on the top, so here I disable the feature for find, and enable it again

**Comment by edannenberg** [comment]
Thanks! Noticed the issue yesterday too, already fixed it locally but it's in my working branch. Will push later today. Also added a file check before find so it wouldn't fail the build if the folders don't exist.

File: bob-core/build-root.sh (hunk 1 above)
Comment (jbergstroem) [inline bob-core/build-root.sh]: haha, sorry; I read it as two `set +e`. Never mind.

**Comment by edannenberg** [comment]
While the `+e/-e` approach won't fail the build it's not ideal. We want the build to fail on errors, just not when the 2 folders are not found.

**Comment by azimut** [comment]
Thanks!, will watch closely for backslashes next time. Closing per above comment.

**Pull request closed without merging**